app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,45 @@
+class FoodsController < ApplicationController
+  def index
+    @foods = Food.where(user: current_user)
+  end
+
+  def show
+    @food = Food.find(params[:id])
+  end
+
+  def new
+    @food = Food.new
+  end
+
+  def create
+    @food = Food.new(food_params)
+
+    respond_to do |format|
+      if @food.save
+        format.html { redirect_to foods_path, notice: 'Food created successfully!' }
+      else
+        format.html { redirect_to foods_path, alert: 'Something went wrong! Food was not created!' }
```

Comment:
[OPTIONAL] I think that we should use `render` instead of `redirect_to` on else statements. This is only optional since it is working perfectly, well done!